administrator/components/com_jedchecker/views/uploads/view.html.php

```diff
@@ -44,13 +50,33 @@ public function display($tpl = null)
 	 */
 	public function getRules()
 	{
-		$rules = array();
-		$files = JFolder::files(JPATH_COMPONENT_ADMINISTRATOR . '/libraries/rules', '\.php$', false, false);
+		$existingRules = array(
```

Comment:
Why are you hard coding this? it causes a serious headache in maintainability.

Reply:
To arrange them in specific order (for sidebar menu). Alternative way is to introduce a `priority` static property, and sort according to that property, but I've postponed this idea until later. I'll try to implement it in a few days.

Reply:
Done